src/get_data_completed_project_package/parsing_url_from_message.py

```diff
@@ -0,0 +1,46 @@
+def parsing_url_from_message(message: str):
+    """
+    Извлекает url из сообщения пользователя.
+    Может удалять query параметры ссылки, так как для очистки
+    c помощью clear_url_of_char "?" используется - replace
+    :param message: Передаем reply сообщение пользователя,
+        чей проект добавляем в итоги месяца.
+    :return:
+    """
+    # Шаблон извлекаемой ссылки
+    sample_url = 'https://github.com'
+    list_message_words = message.split()
+
+    for word in list_message_words:
+        if len(word) >= len(sample_url):
+            slice_word = word[:len(sample_url)]
+            if slice_word == sample_url:
+                result_url = __clear_url_of_char(word)
+                return result_url
+        else:
+            continue
```

Comment:
Можно сделать инверсию 

```suggestion
        if len(word) <= len(sample_url):
            continue
        slice_word = word[:len(sample_url)]
        if slice_word == sample_url:
            result_url = __clear_url_of_char(word)
            return result_url  
```